specifyweb/context/views.py

```diff
@@ -602,3 +609,24 @@ def api_endpoints(request):
 def api_endpoints_all(request):
     """Returns a JSON description of all endpoints."""
     return JsonResponse(generate_openapi_for_endpoints(True))
+
+@require_GET
+@never_cache
+def set_language(request, language: str):
```

Comment:
This should be a POST method and use a request parameter instead of a url path parameter. As a general rule, GET methods shouldn't "change" anything.

Reply:
This particular endpoint wasn't used. I have just removed it (ac2037b5)
But I see what you are saying about GET not being supposed to make changes.